presto-docs/src/main/sphinx/develop/delete-and-update.rst

```diff
@@ -0,0 +1,178 @@
+====================================
+Supporting ``DELETE`` and ``UPDATE``
+====================================
+
+The Presto engine provides APIs to support row-level SQL ``DELETE`` and ``UPDATE``.
+To implement ``DELETE`` or ``UPDATE``, a connector must layer an ``UpdatablePageSource``
+on top of the connector's usual ``ConnectorPageSource``, and define ``ConnectorMetadata``
+methods to get a "rowId" column handle; to start the operation; and to finish the operation.
+
+``DELETE`` and ``UPDATE`` Data Flow
+===================================
+
+``DELETE`` and ``UPDATE`` have a similar flow:
+
+* For each split, the connector will create an ``UpdatablePageSource`` instance, layered over the
+  connector's ``ConnectorPageSource``, to read pages on behalf of the Presto engine, and to
+  write deletions and/or updates to the underlying data store.
```

Comment:
```suggestion
  write deletions or updates to the underlying data store.
```
From the [GitLab recommended word list entry for and/or](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#andor), 

"Instead of and/or, use or or rewrite the sentence to spell out both options."